python/shotgun_model/data_handler_nav.py

```diff
@@ -30,7 +31,7 @@ class ShotgunNavDataHandler(ShotgunDataHandler):
     _SG_PATH_FIELD = "path"
     _SG_PARENT_PATH_FIELD = "parent_path"
 
-    def __init__(self, root_path, seed_entity_field, entity_fields, cache_path, parent):
+    def __init__(self, root_path, seed_entity_field, entity_fields, cache_path):
```

Comment:
ah! as per Josh's suggestion, the data handler no longer inherits from qobject. it's just a normal py-class.